packages/squiggle-lang/__tests__/Distributions/SampleSetDist_ToPointSet_test.res

```diff
@@ -0,0 +1,22 @@
+open Jest
+open Expect
+
+describe("Converting from a sample set distribution", () => {
+  test("Should be normalized", () => {
+    let outputXYShape = SampleSetDist_ToPointSet.Internals.KDE.normalSampling(
+      [1., 2., 3., 3., 4., 5., 5., 5., 6., 8., 9., 9.],
+      50,
+      2,
+    )
+    let c: PointSetTypes.continuousShape = {
+      xyShape: outputXYShape,
+      interpolation: #Linear,
+      integralSumCache: None,
+      integralCache: None,
+    }
+    let fullShape = Continuous.updateIntegralCache(Some(Continuous.T.integral(c)), c)
+    let endY = Continuous.T.integralEndY(fullShape)
```

Comment:
can we used the ``isNormalized`` function? (I thought we had one in continuous distribution; and if we don't, we should have one there)